core/src/test/scala/org/broadinstitute/dsde/rawls/model/SpendReportingModelSpec.scala

```diff
@@ -0,0 +1,199 @@
+package org.broadinstitute.dsde.rawls.model
+
+import bio.terra.profile.model.SpendReport
+import bio.terra.profile.model.SpendReportingForDateRange.CategoryEnum
+import bio.terra.profile.model.{SpendReportingForDateRange => SpendReportingForDateRangeBPM}
+import bio.terra.profile.model.{SpendReportingAggregation => SpendReportingAggregationBPM}
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+import org.scalatest.flatspec.AnyFlatSpecLike
+import org.scalatest.matchers.must.Matchers.have
+import org.scalatest.matchers.should.Matchers.{convertToAnyShouldWrapper, equal}
+
+import scala.jdk.CollectionConverters._
+
+class SpendReportingModelSpec extends AnyFlatSpecLike {
+  object TestData {
+    val defaultCurrency = "USD"
+    val defaultAzureCredits = "0" // always 0 for Azure
+
+    def buildBPMEmptyReport: SpendReport = {
+      // some period
+      val from = DateTime.now().minusMonths(2)
+      val to = from.plusMonths(1)
+
+      // BVM return following spendReport in case specific data doesn't exist
```

Comment:
For GCP, Rawls returns a 404 when there's no data. I don't really have a preference between a 404 or returning an empty report like BPM does today, but they should at least be consistent. What do you think?

Making them consistent falls outside the scope of this PR in my opinion, but wanted to note the inconsistency

Reply:
Yes, I agree. I think it would be definitely better to keep responses consistent. It probably requires changes not only in Rawls, but in BPM too.